app/controllers/state_file/questions/questions_controller.rb

```diff
@@ -24,22 +24,21 @@ def current_intake
       end
 
       def question_navigator
-        case params[:us_state]
-        when 'az'
-          Navigation::StateFileAzQuestionNavigation
-        when 'ny'
-          Navigation::StateFileNyQuestionNavigation
+        @navigator ||= "Navigation::StateFile#{state_code.titleize}QuestionNavigation".constantize
+      end
+
+      def state_code
+        state_code_ = params['us_state']
+        unless StateFileBaseIntake::STATE_CODES.include?(state_code_)
+          raise StandardError state_code_
         end
+        state_code_
```

Comment:
This method takes in and validates the state code parameter (So we can constantize / send without fear of code injection)

Reply:
Nice

Reply:
Do you think just passing in the state code is enough? Maybe raising something like `raise StandardError, "State code `#{state_code_}` not supported!"` might make it easier to find the error when we see it in Sentry.